### PR TITLE
WIP - Replacing defaultcontent by migrate

### PR DESCRIPTION
**Pull request by davidferlay**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -105,6 +100,10 @@ localize:
 	$(call php, drush locale:import:all /var/www/html/translations/ --type=customized --override=all)
 	@echo "Localization finished"
 
+## Import structural/default/test/real content
+content:
+	$(call php, drush migrate XXX)
```
Comment (andypost) [inline Makefile]: Xxx needs config + readme

File: Makefile [hunk 2]
```diff
@@ -30,11 +30,11 @@ php = docker-compose exec -T --user $(CUID):$(CGID) php ${1}
 php-0 = docker-compose exec -T php ${1}
 
 ## Full site install from the scratch
-all: | provision back front si localize hooksymlink info
+all: | provision back front si localize content hooksymlink info
 # Install for CI deploy:review. Back & Front tasks are run in a dedicated previous step in order to leverage CI cache
-all_ci: | provision si localize hooksymlink info
+all_ci: | provision si localize content info
```
Comment (andypost) [inline Makefile]: Why removed here?

Comment (davidferlay) [reply Makefile]: (Unrelated to goal of this PR) Makes no sense to add git hooks using symlink in RA.

**Comment by davidferlay** [comment]
merge reverted in master and PR re-opened

**Pull request merged**